io.openems.edge.io.shelly/src/io/openems/edge/io/shelly/shelly25/IoShelly25.java

```diff
@@ -123,6 +168,164 @@ public default void setRelay1(boolean value) throws OpenemsNamedException {
 		this.getRelay1Channel().setNextWriteValue(value);
 	}
 
+	/**
+	 * Gets the Channel for {@link ChannelId#RELAY_1_OVERTEMP}.
+	 *
+	 * @return the Channel
+	 */
+	public default BooleanWriteChannel getRelay1OvertempChannel() {
```

Comment:
The purpose of those methods is to allow easy access of channel values in (complicated) code. If you don't use them anywhere, it's not required to add them.